_sass/_layout.scss

```diff
@@ -81,10 +86,25 @@
 
 .tm-content {
   border-radius: 2px;
-  padding: 60px;
+  padding: 20px;
   margin-bottom: 60px;
 
+  @media (min-width: 480px) {
+    padding: 40px 28px;
+  }
+  @media (min-width: 768px) {
+    padding: 60px;
+  }
+
   h3 {
     margin-top: 48px;
   }
 }
+
+/**
+ * Material Lite Overrides
+ */
+
+.mdl-layout {
+  height: auto;
```

Comment:
This was necessary to fix the native 'scroll-to-id' effect. It is a known issue on Material Lite.